code/modules/clothing/spacesuits/plasmamen.dm

```diff
@@ -41,16 +41,23 @@
 	icon_state = "plasmaman-helm"
 	item_state = "plasmaman-helm"
 	strip_delay = 80
+	flash_protect = 0
```

Comment:
!

Reply:
This change is legit and not unlisted, but if the masses argue against it, I can undo this bit.